src/presentation/components/note-list/NoteListItem.vue

```diff
@@ -46,56 +39,61 @@ const noteTitle = computed(() => {
     return firstNoteBlock.data.text.slice(0, limitCharsForNoteTitle);
   }
 });
-
 /**
- * Get the subtitle depending on the type of block from Note
+ * Get the update date of the Note
  */
-const noteSubTitle = computed(() => {
-  const secondNoteBlock = props.note.content.blocks[1];
+const noteUpdatedAt = new Date(String(props.note.updatedAt));
 
-  if (secondNoteBlock) {
-    switch (secondNoteBlock.type) {
-      case 'header':
-      case 'paragraph':
-        return secondNoteBlock.data.text.slice(0, limitCharsForNoteSubTitle);
-      default:
-        return '';
-    }
-  } else {
-    return '';
-  }
-});
-/**
- * Get the creation date of the Note
- */
-const noteCreatedDate = new Date(String(props.note.createdAt)).toLocaleDateString();
+const currentDate = new Date();
+
+const timeDifference = currentDate.getTime() - noteUpdatedAt.getTime();
+const secondsDifference = Math.floor(timeDifference / 1000);
+const minutesDifference = Math.floor(secondsDifference / 60);
+const hoursDifference = Math.floor(minutesDifference / 60);
+const daysDifference = Math.floor(timeDifference / (1000 * 3600 * 24));
+const weeksDifference = Math.floor(daysDifference / 7);
+const monthsDifference = Math.floor(daysDifference / 30);
+const yearsDifference = Math.floor(daysDifference / 365);
+let formattedUpdatedAt: string;
+
+if (secondsDifference < 60) {
+  formattedUpdatedAt = 'Just now';
+} else if (minutesDifference < 60) {
+  formattedUpdatedAt = `${minutesDifference} ${minutesDifference === 1 ? 'minute' : 'minutes'} ago`;
+} else if (hoursDifference < 24) {
+  formattedUpdatedAt = `${hoursDifference} ${hoursDifference === 1 ? 'hour' : 'hours'} ago`;
+} else if (daysDifference < 7) {
+  formattedUpdatedAt = `${daysDifference} ${daysDifference === 1 ? 'day' : 'days'} ago`;
+} else if (daysDifference < 30) {
+  formattedUpdatedAt = `${weeksDifference} ${weeksDifference === 1 ? 'week' : 'weeks'} ago`;
+} else if (daysDifference < 365) {
+  formattedUpdatedAt = `${monthsDifference} ${monthsDifference === 1 ? 'month' : 'months'} ago`;
+} else {
+  formattedUpdatedAt = `${yearsDifference} ${yearsDifference === 1 ? 'year' : 'years'} ago`;
+}
```

Comment:
Duplication of logic on working with dates?

Reply:
Oh, I think that we might even delete this whole component in this folder, as we are developing and using the one from codex-ui, right?

Reply:
correct